src/main/webapp/app/js/pureJSCalendar.js

```diff
@@ -0,0 +1,274 @@
+var pureJSCalendar = (function () {
```

Comment:
Please include a reference to the source of all third-party code you use in your project.